### PR TITLE
added dear_vehicle_owner meme keymap to planck keyboard

### PR DESCRIPTION
**Pull request by BeckTimothy**
## Description

We've been trying to reach you concerning your vehicle's extended warranty. You should've received a notice in the mail about your car's extended warranty eligibility. Since we've not gotten a response, we're giving you a final courtesy call before we close out your file. Press 2 to be removed and placed on our do-not-call list. To speak to someone about possibly extending or reinstating your vehicle's warranty, press 1 to speak with a warranty specialist.

## Types of Changes

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [ ] Keyboard (addition or update)
- [x] Keymap/layout/userspace (addition or update)
- [ ] Documentation

## Issues Fixed or Closed by This PR

n/a

## Checklist

- [x] My code follows the code style of this project: [**C**](https://docs.qmk.fm/#/coding_conventions_c), [**Python**](https://docs.qmk.fm/#/coding_conventions_python)
- [x] I have read the [**PR Checklist** document](https://docs.qmk.fm/#/pr_checklist) and have made the appropriate changes.
- [ ] My change requires a change to the documentation.
- [x] I have updated the documentation accordingly.
- [x] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [x] I have added tests to cover my changes.
- [x] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

File: keyboards/planck/keymaps/dear_vehicle_owner/config.h
Comment (drashna) [inline keyboards/planck/keymaps/dear_vehicle_owner/config.h:1]: Could you add a GPL2+ compatible license header here? 

For instance: 
https://github.com/qmk/qmk_firmware/blob/28ff51175b6d45ce5a8dbfe7ed6e7a12df3bc8a8/quantum/template/base/keyboard.c#L1-L15

File: keyboards/planck/keymaps/default/keymap.c
Comment (drashna) [inline keyboards/planck/keymaps/default/keymap.c:1]: Could you revert the changes to the default keymap here?

**Review by drashna — CHANGES_REQUESTED** [review]

File: keyboards/planck/keymaps/dear_vehicle_owner/config.h
Comment (BeckTimothy) [reply keyboards/planck/keymaps/dear_vehicle_owner/config.h:17]: resolved in 606d158

File: keyboards/planck/keymaps/default/keymap.c
Comment (BeckTimothy) [reply keyboards/planck/keymaps/default/keymap.c]: Yikes! Yeah. I modified the wrong directory!

reverted in 4918ec0

**Review by drashna — APPROVED** [review]

**Comment by BeckTimothy** [comment]
So when does this get merged, or is there something I need to do?

**Comment by drashna** [comment]
Generally, we want two approvals on any PR before it gets merged. 

Also, it can take a while for a PR to get reviewed, as we do this in our free time.

**Review by spidey3 — APPROVED** [review]

**Pull request merged**
